lib/roo_on_rails/routemaster/publisher.rb

```diff
@@ -33,6 +33,10 @@ def url
         raise NotImplementedError
       end
 
+      def async?
+        false
```

Comment:
The default in `routemaster-client` is `false` so I thought I'd keep the same default otherwise it might be confusing.